src/enums/button-selector.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum ButtoDisplayoption {

```

Comment:
Is this typo intentional?

Reply:
it accident with intentional that I typed wrong but already copy paste all file.

Reply:
```suggestion
export enum ButtonDisplayOption {
```